PHP 8 compatibility issue

Looks like the BLT launcher has the same issue the Drush launcher had with `padraic/phar-updater` being abandoned and not compatible with PHP 8: https://github.com/drush-ops/drush-launcher/issues/84. This is preventing the BLT launcher from being run on any envs with PHP 8.

Seeing this issue trying to run blt on PHP 8:
```
Box Requirements Checker
========================

> Using PHP 8.0.8
> PHP is using the following php.ini file:
  /usr/local/php8.0/etc/cli/php.ini

> Checking Box requirements:
  ..E.........

                                                                                
 [ERROR] Your system is not ready to run the application.                       
                                                                                

Fix the following mandatory requirements:
=========================================

 * The package "padraic/humbug_get_contents" requires the version "^5.3 || ^7.0
   || ^7.1 || ^7.2" or greater.
```

--- a/composer.json
+++ b/composer.json
@@ -5,7 +5,7 @@
     "require": {
         "webflo/drupal-finder": "^1.0",
         "webmozart/path-util": "^2.3",
-        "padraic/phar-updater": "^1.0",
+        "laravel-zero/phar-updater": "dev-main",
         "composer/xdebug-handler": "^1.0"
     },
     "authors": [
@@ -30,7 +30,7 @@
     },
     "scripts": {
         "box-install": [
-            "curl -O -L https://github.com/box-project/box/releases/download/3.8.5/box.phar"
+            "curl -O -L https://github.com/box-project/box/releases/download/3.11.1/box.phar"
         ],
         "box-compile": [
             "php box.phar compile"

